internal/configuration/validator/const.go

```diff
@@ -125,8 +129,9 @@ var validKeys = []string{
 }
 
 var specificErrorKeys = map[string]string{
-	"logs_file_path": "config key replaced: logs_file is now log_file",
-	"logs_level":     "config key replaced: logs_level is now log_level",
+	"logs_file_path":   "config key replaced: logs_file is now log_file",
+	"logs_level":       "config key replaced: logs_level is now log_level",
+	"read_buffer_size": "config key moved: read_buffer_size is now under a new block called server, making it server.read_buffer_size",
```

Comment:
Is this actually necessary?
It didn't actually exist as a key in Production before this change, presumably you would be catering towards a single person having tested this during the design.

Reply:
That's true, I did it mainly due to anyone testing. But should only be the one, so you're right.